repo2docker/buildpacks/guix/__init__.py

```diff
@@ -0,0 +1,74 @@
+"""BuildPack for guix environments"""
+import os
+
+from ..base import BuildPack, BaseImage
+
+
+class GuixBuildPack(BaseImage):
+    """A guix Package Manager BuildPack"""
+
+    def get_path(self):
+        """Return paths to be added to PATH environment variable"""
+        return super().get_path() + ["/home/${NB_USER}/.guix-profile/bin"]
+
+
+    def get_build_scripts(self):
+        """
+        Install guix package manager version 1.3.0.x86_64-linux, using
+        an unmodified installation script found at
+        https://git.savannah.gnu.org/cgit/guix.git/plain/etc/guix-install.sh
+        """
+        return super().get_build_scripts() + [
+            (
+                "root",
+                """
+                yes | BIN_VER=1.3.0.x86_64-linux \
+                bash /home/${NB_USER}/.local/bin/guix-install.bash
```

Comment:
Unfortunately this file appears to be GPL, and so can not be included in the repo. Can we fetch it at build time, just like we do for nix?

Reply:
Done, guix-install.bash is not under GPL anymore, a pinned installation script is now fetched at build time!